annotations/test_utils.py

```diff
@@ -3,6 +3,7 @@
 from .test_models import BaseTestCase
 
 
+# TODO bram: extend unit tests
```

Comment:
Yes, would be nice to have a test for the captcha, showing that it will appear for unauthenticated users! :-)